include/experimental/__p1673_bits/blas2_matrix_vector_product.hpp

```diff
@@ -66,6 +66,7 @@ template<class ElementType_A,
          extents<>::size_type ext_y,
          class Layout_y,
          class Accessor_y>
+  requires (Layout_A::is_always_unique())
```

Comment:
I think we discussed this one -- `Layout_A::template mapping<extents<numRows_A, numCols_A>>::is_always_unique()`.